conans/client/conan_command_output.py

```diff
@@ -117,6 +118,19 @@ def print_search_packages(self, search_info, reference, packages_query, table,
             printer.print_search_packages(search_info, reference, packages_query,
                                           outdated=outdated)
 
+    def print_revisions(self, ref, revisions, remote_name=None):
```

Comment:
It is already a string

```suggestion
    def print_revisions(self, reference, revisions, remote_name=None):
```